Rider/app/src/main/res/layout/activity_main.xml

```diff
@@ -1,27 +1,311 @@
+                        android:layout_marginBottom="20dp"
+                        android:text="@string/driverNum"
+
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintLeft_toRightOf="@id/phone">
+
+                    </TextView>
+
+
+                </androidx.constraintlayout.widget.ConstraintLayout>
+
+
+            </androidx.cardview.widget.CardView>
+
+            <TextView
+                android:id="@+id/upcomingRides"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="18dp"
+                android:text="@string/upcomingRides"
+                android:textColor="@color/darkerGrey"
+                android:textFontWeight='"600"'
+                android:textSize="@dimen/rideTextSize"
+                android:textStyle="bold"
+
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/nextRideCard"></TextView>
+
+            <HorizontalScrollView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:id="@+id/upcomingRidesView"
+
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/upcomingRides">
+
+                <include
+                    android:id="@+id/urcard1"
+                    layout="@layout/upcoming_rides"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginLeft="@dimen/margin"
+                    android:layout_marginTop="@dimen/margin"
+                    android:layout_marginRight="@dimen/margin">
+
+                </include>
+
+            </HorizontalScrollView>
+
+            <TextView
+                android:id="@+id/rideHist"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="18dp"
+                android:text="@string/rideHist"
+                android:textColor="@color/darkerGrey"
+                android:textFontWeight='"600"'
+                android:textSize="@dimen/rideTextSize"
+                android:textStyle="bold"
+
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/upcomingRidesView">
+
+            </TextView>
+
+            <ScrollView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="5dp"
+
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/rideHist">
```

Comment:
@+id (these all seem to be on the same layout_constraint, so if I'm wrong here lmk)